POS78 keyboard folder added (#11024)
qmk/qmk_firmware: keyboards/pos78/readme.md

# pos78

> https://imgur.com/a/DimaAL5

Reviewer: ```suggestion
![](https://i.imgur.com/ml5cPDS.png)
```

> # pos78

Reviewer 2: Can you reformat this file to match our [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)?

Author: One of the other reviewers suggested this format...

Reviewer 3: Yeah that was my bad, sorry.

* Keyboard Maintainer: [smssmssms](https://github.com/smssmssms)
* Hardware Supported: ATMEGA 32U4 (Adafruit Itsy Bitsy 5v) 
* Hardware Availability: 1980-90's version of the [Partner Tech Corp KB-78](https://www.partner.com.tw/wp-content/uploads/2019/09/KB-78-G-78-1-190911.pdf); a 78 key Point-Of-Sale ortholinear keyboard conversion
  

> 
> Example on how to compile and flash the keyboard [after setting up the build environment](https://docs.qmk.fm/#/getting_started_build_tools))
>     
>     qmk flash -kb pos78 -km default
> 
> Only want to compile the firmware?
>     
>     qmk compile -kb pos78 -km default
> 
> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. 
> 
> Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make pos78:default

Flashing example for this keyboard:

    make pos78:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
